src/client/src/App/Credit/common/RfqTicketHeader.ts

```diff
@@ -67,6 +68,12 @@ export const DirectionLabel = styled.div<WithDirection & WithTerminated>`
     -ms-transform: skewX(-24deg);
     transform: skewX(-24deg);
   }
+
+  div {
+    width: ${({ direction }) =>
+      direction === Direction.Buy ? "45px" : "50px"};
```

Comment:
```suggestion
    width: "50px"
```

Reply:
seems like 50 works for both?

Reply:
To me it's slightly off but I guess nobody would care about such a small difference.